.ci/build_demo.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash -e
+
+# Builds a new demo instance using Docker Compose.
+# Uses URLs <demo_name>.loki.shopsys.com and 2.<demo_name>.loki.shopsys.com
```

Comment:
You're right, I've thought about it in the beginning, but there were regexes at the time and I wanted to avoid escaping the dots. In the meantime I've removed the regexes so now it's pretty easy to parameterized (see 2b93c6f7d)